### ISSUE TITLE
Add option for handling submodules

### ISSUE THREAD

**Issue by rafecolton** [rafecolton/kamino#3]
What is the best way to do this?  Should `git clone` also pull submodules (i.e. `--recursive`)?

**Comment by meatballhat** [comment]
I would cargo cult whatever Travis CI is doing at the beginning of every
build.

**Comment by rafecolton** [comment]
Yeah that's kind of what prompted this inquiry.  Travis automatically pulls submodules, so maybe `--recursive` should be the default with an option not to pull submodules?  That option would have to be passed all the way through the build server of course

**Comment by meatballhat** [comment]
whatever Travis is doing.  do that.  recursive.  automatic.  whatever.

:heart_eyes_cat:

**Comment by jszwedko** [comment]
:+1: to `--recursive`
